localstack/services/events/provider.py

```diff
@@ -143,6 +156,21 @@ def format_event(event: PutEventsRequestEntry, region: str, account_id: str) ->
     return formatted_event
 
 
+def get_bus_or_rule(arn: Arn) -> ResourceType:
+    parsed_arn = parse_arn(arn)
+    service = parsed_arn["resource"].split("/", 1)[0]
```

Comment:
`service` could be misleading here given that `parsed_arn["service"]` exists. `resource_type` is more accurate here following the ARN definition: https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html#arns-syntax

Reply:
fixed in: https://github.com/localstack/localstack/pull/10840/commits/2ccecbc2d1b61126a3528c7296862365c04b0126